### PR TITLE
Update IFTTT integration and new IFTTT for Google Assistant

### PR DESCRIPTION
**Pull request by Rocklita**
Created a new page for Google Assistant using IFTTT and updated/cleaned up the other IFTTT page. First time doing this so please let me know if I'm stepping on anyone's toes or doing this wrong.

### CODE REVIEW

File: docs/docs/walkthrough/phase-4/ifttt-google-assistant.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# IFTTT Integration for Google Assistant
+
+Want to use Google Assistant to set your carb intake, insulin given, eating soon, etc.? Follow the instructions below to use IFTTT with the Google Assistant service.
+
+## Prerequisites
+
+* Get an IFTTT.com account
+* Make sure you have [Maker](https://www.ifttt.com/maker) connected to your IFTTT account.
+* Make sure you have [Google Assistant] (https://www.ifttt.com/google_assistant) connected to your IFTTT account.
```
Comment (scottleibrand) [inline docs/docs/walkthrough/phase-4/ifttt-google-assistant.md]: Does the space between ] and ( need to be removed here?

Comment (Rocklita) [inline docs/docs/walkthrough/phase-4/ifttt-google-assistant.md]: Hmmm.... I'm not really sure!! It shows good in the preview, but I see what you're saying. I'm ok with removing it. Do I have to do another pull request?
